transpiler/src/ast/core/intermediate-ast/nodes/Domain/DomainCreateNode.ts

```diff
@@ -33,6 +35,20 @@ export class DomainCreateNode extends IntermediateASTNode {
     return [parameterNode];
   }
 
+  public getBitloopsPrimaryType(): BitloopsPrimaryTypeNode | null {
+    return this.getChildNodeByType<BitloopsPrimaryTypeNode>(
+      BitloopsTypesMapping.TBitloopsPrimaryType,
+    );
+  }
+
+  public getReturnOkErrorType(): ReturnOkErrorTypeNode | null {
```

Comment:
why do we need all these methods? Domain create only has ReturnOkErrorTypeNode, correct?